<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>plan_creation_window</class>
 <widget class="QMainWindow" name="plan_creation_window">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>690</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>690</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>690</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Plan Creator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(211, 215, 207);</string>
  </property>
  <widget class="QWidget" name="body_widget">
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(211, 215, 207);
border: 0px solid;</string>
   </property>
   <widget class="QPushButton" name="close_plan_pbutton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_plan_pbutton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>650</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="new_plan_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Treatment plan label</string>
    </property>
   </widget>
   <widget class="QLabel" name="new_plan_ref_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Reference plan</string>
    </property>
   </widget>
   <widget class="QComboBox" name="new_plan_ref_cbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>None</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="new_plan_ledit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="new_img_path_tbutton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Add the CT and segmentation data from a folder</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/black_icons/icons_black/file-plus.svg</normaloff>:/black_icons/icons_black/file-plus.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="new_img_path_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Imaging path</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_img_path_ledit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
   </widget>
   <widget class="QToolButton" name="new_dose_path_tbutton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>510</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>31</width>
      <height>31</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="toolTip">
     <string>Add the dose-influence matrix from a folder</string>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <property name="icon">
     <iconset resource="../assets/resources.qrc">
      <normaloff>:/black_icons/icons_black/file-plus.svg</normaloff>:/black_icons/icons_black/file-plus.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>18</width>
      <height>18</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="new_dose_path_ledit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>30</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
   </widget>
   <widget class="QLabel" name="new_dose_path_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Dose matrix path</string>
    </property>
   </widget>
   <widget class="QLabel" name="new_modality_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>163</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>163</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>163</width>
      <height>31</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Modality</string>
    </property>
   </widget>
   <widget class="QComboBox" name="new_modality_cbox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>163</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>163</width>
      <height>30</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>163</width>
      <height>30</height>
     </size>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
    </property>
    <item>
     <property name="text">
      <string>photon</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>proton</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="new_dose_res_label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>560</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>31</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="toolTip">
     <string>Size of the dose grid in [mm] per dimension</string>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Dose grid resolution [mm]</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>333</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="new_img_res_layout">
     <item>
      <widget class="QLabel" name="new_img_res_sublabel_xyz">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>[x, y, z] : </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_img_res_hspacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="new_img_res_bracket_left">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_img_res_hspacer_between_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="new_img_res_ledit_x">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="new_img_res_comma_left">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="new_img_res_ledit_y">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="new_img_res_comma_right">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="new_img_res_ledit_z">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string>None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_img_res_hspacer_between_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="new_img_res_bracket_right">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_img_res_hspacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="new_img_res_label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>380</y>
      <width>347</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>25</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>16777215</width>
      <height>25</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid;</string>
    </property>
    <property name="text">
     <string>Target imaging resolution [mm]</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>600</y>
      <width>333</width>
      <height>32</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="new_dose_res_layout">
     <item>
      <widget class="QLabel" name="new_dose_res_sublabel_xyz">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>[x, y, z] : </string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_dose_res_hspacer_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="new_dose_res_bracket_left">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>[</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_dose_res_hspacer_between_left">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="new_dose_res_ledit_x">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="new_dose_res_comma_left">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="new_dose_res_ledit_y">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="new_dose_res_comma_right">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>,</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="new_dose_res_ledit_z">
       <property name="minimumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>45</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(238, 238, 236);
border: 1px solid;
border-color: rgb(186, 189, 182);</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_dose_res_hspacer_between_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="new_dose_res_bracket_right">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 0px solid;</string>
       </property>
       <property name="text">
        <string>]</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="new_dose_res_hspacer_right">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>30</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../assets/resources.qrc"/>
 </resources>
 <connections/>
</ui>
